app/js/components/tabs/tabcomponents/savedComponent.jsx

```diff
@@ -26,13 +26,15 @@ class SavedComponent extends React.Component {
       definitionDeleteJobs: []
     };
     // cohort related methods
-    this.getLastSavedCohorts = this.getLastSavedCohorts.bind(this);
+    // TODO: To be added when it's possible to sort the results fetched
+    // this.getLastSavedCohorts = this.getLastSavedCohorts.bind(this);
     this.searchSavedCohorts = this.searchSavedCohorts.bind(this);
     this.deleteCohort = this.deleteCohort.bind(this);
     this.downloadCohort = this.downloadCohort.bind(this);
     this.viewCohort = this.viewCohort.bind(this);
     // definition related methods
-    this.getLastSavedDefinitions = this.getLastSavedDefinitions.bind(this);
+    // TODO: To be added when it's possible to sort the results fetched
+    // this.getLastSavedDefinitions = this.getLastSavedDefinitions.bind(this);
```

Comment:
@dkayiwa I left all the commented code and put the TODO tag because they are still valid except that we currently can't query for a sorted results. Hopefully this would be possible in the future and all that would be needed would be to uncomment the code. What do you think ?